send-avs-logs-to-log-server/manifest.json

```
@@ -0,0 +1,55 @@
+{
+    "title": "Azure VMware Solution: Export logs to log management solution",
+    "description": "This workflow template triggers when log messages from your Azure VMware Solution private cloud are available in Event Hub. Those messages are then processed and send to the endpoint of the log management tool you specificy, such as VMware Aria Operations for Logs.",
+    "prerequisites": "Before you can use this template, you need to have an Azure VMware Solution private cloud provisioned and be sending your private cloud log messages to an Event Hub namespace. For more information on setting up your Azure VMware Solution private cloud, see how to [Deploy and Configure Azure VMware Solution](https://learn.microsoft.com/en-us/azure/azure-vmware/deploy-azure-vmware-solution?tabs=azure-portal). For details on how to send your logs to Azure Event Hub, see [Configure VMware Syslogs: Stream to Azure Event Hub](https://learn.microsoft.com/en-us/azure/azure-vmware/configure-vmware-syslogs#stream-to-microsoft-azure-event-hubs).",
```

- Need to use go.microsoft.com links, not hardcoded learn.microsoft.com links.
- Doc titles use sentence case, not title case. We don't capitalize nouns unless they appear in brand or product names.

```suggestion
    "prerequisites": "Before you use this template, you need to have an Azure VMware Solution private cloud instance and have your private cloud log messages sent to an Azure Event Hubs namespace. For more information about setting up Azure VMware Solution private cloud, see [Deploy and configure Azure VMware Solution](https://go.microsoft.com/fwlink/?linkid=2305015). For more information about sending logs to Azure Event Hubs, see [Configure VMware syslogs - Stream to Azure Event Hubs](https://go.microsoft.com/fwlink/?linkid=2305111).",
```